services/booking/create_order.rb

```diff
@@ -0,0 +1,34 @@
+class BookingService
+  class CreateOrder < ApplicationService
+    delegate :params_valid?, to: :policy
+
+    def initialize(params)
+      @params = params
+    end
+
+    def call
+      create_order if params_valid?
+    end
+
+    def policy
```

Comment:
Move this under `private` as well as `create_order` method
Service Interface should expose only `call` method most of the times